### DISCUSSION TITLE
Airgap mode only in some nodes of my distributed Deployment

### DISCUSSION THREAD

**Discussion by Angelozinna96** [Security-Onion-Solutions/securityonion#3607] [Unsupported Versions] [answered]
Hi everyone,
I'm planning to use a distributed deployment with a manager node, 2 search nodes and a forward node.
However, I have the possibility of giving internet access in a limited way (just a few hours a week) to only a single node.
I was wondering if I could use a distributed deployment with all my nodes in airgap mode except the manager node or using the manager node as a proxy/gateway for all the other nodes.
What would you suggest?

**Comment by weslambert** [comment, marked as answer]
I would suggest giving your manager internet access, then using the manager proxy option for the other nodes to use the manager as the proxy for updates.  You can select this option when running setup for the manager and other nodes.

**Reply by Angelozinna96** [reply]
Ok, I'll do it so, thank you very much!!
